code/lib/codemod/src/transforms/find-implicit-spies.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-underscore-dangle */
+import type { FileInfo } from 'jscodeshift';
+import { loadCsf } from '@storybook/csf-tools';
+import type { BabelFile } from '@babel/core';
+import * as babel from '@babel/core';
+
+function findImplicitSpies(path: babel.NodePath, file: string) {
+  path.traverse({
+    Identifier: (identifier) => {
+      if (/^on[A-Z].*/.test(identifier.node.name)) {
+        console.warn(identifier.buildCodeFrameError(`${file} Possible implicit spy found`).message);
```

Comment:
I see your point about why this is only a "possible implicit spy". Shouldn't we add an extra check to determine whether the spy is actually explicit by parsing the `args` object and whether an explicit mock is assigned?

Reply:
Furthermore, the console might be spammed about these warnings. Let's say in huge Storybooks, where the play function is also used more or less everywhere, these warning might be triggered a ton of time. 

So a couple of suggestions:

1. Should we introduce a counter only to log a maximum amount of warning messages?
2. The rest could be written into a separate file. Then people might be able to go through each finding step by step.